pkgs/applications/video/handbrake/default.nix

```diff
@@ -37,15 +37,13 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [
     cmake python2 pkgconfig yasm autoconf automake libtool m4
```

Comment:
Are you sure autoconf and automake are still needed with cmake/ninja?

Reply:
I have old mild knowledge of C++ builds, I used to C++ years ago, when Ninja was not around.

So:
If I add `ninja` to `nativeBuildInputs` system tries to use it and fails:
```
Build is configured!
You may now cd into ./build and run make (/nix/store/h9w5yn61phv86iirl273bpw5ksq98fzz-gnumake-4.2.1/bin/make).
building
no build.ninja, doing nothing
glibPreInstallPhase
installing
install flags: install
ninja: error: loading 'build.ninja': No such file or directory
builder for '/nix/store/38fygqhfawrhni6f166dl5ds8akhdgyb-handbrake-1.0.7.drv' failed with exit code 1
```

If I while adding `ninja` also remove `autoconf` and `automake` it just falls flat the same way.

Obviously there is more steps needed here. Or HandBrake dudes do it the old way.

So from the hip, it did not went.

From first look at different NixPkgs sources - there is all that Ninja needs, to switch to it.

In a time I can try to understand more of what are you talking about or take your guidance.

It can be next PRs. Or in this one if you think so.

Reply:
I can't find any CMakeLists.txt file in the source distribution so you might be able to rip out cmake.